app-backup/restic/restic-0.4.0.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="restic is a backup program that is fast, efficient and secure"
+HOMEPAGE="https://restic.github.io/"
+SRC_URI="https://github.com/restic/restic/archive/v${PV}.tar.gz"
```

Comment:
For simplicity, let's [rename](https://devmanual.gentoo.org/ebuild-writing/variables/index.html#renaming-sources) this tarball to `${P}.tar.gz`. i.e.
```
SRC_URI="https://github.com/restic/restic/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```